src/TodoList.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react'
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
+import './App.css'
+
+const list =[
+    {
+      id: 1,
+      name : 'create an agreement with utility companies',
+      deadline: 'wednesday'
+    },
+    {
+      id: 2,
+      name : 'change driver license',
+      deadline: 'tuesday'
+    },
+    {
+      id: 3,
+      name : 'meet with friend regarding the work',
+      deadline: 'thursday'
+    },
+    {
+      id: 4,
+      name : 'Set up an appointment to connest the internet',
+      deadline: 'thursday'
+    }
+  ]
+
+function TodoList()
+{
+    return (
+        <div>
+            <ul>
+                {list.map(function (item) {
+                    return <li key={item.id}>{item.name}</li>
+                }
+      )}
+      </ul>

```

Comment:
The current formatting has some braces that do not close at the appropriate levels, which can make the code harder to read and debug. For example: the closing braces for the map function and its callback are not aligned with the indentation level of the opening braces, making the structure appear inconsistent.
Please align the braces properly to clearly indicate where each block of code begins and ends. By ensuring that the braces close at the same indentation level as their corresponding opening braces, the code will be easier to understand and maintain.
<img width="484" alt="Screenshot 2024-11-16 at 11 00 22" src="https://github.com/user-attachments/assets/67197d84-e3d7-40da-8c0c-31f46a13d9a3">